html/ops/db_update.php

```diff
@@ -1107,6 +1107,7 @@ function update_4_19_2018() {
 	add column previous_email_addr varchar(254) not null default '',
 	add column email_addr_change_time double not null default 0
     ");
+    do_query("alter table user add index user_email_time (email_addr_change_time)");
```

Comment:
I think this needs a semi-colon at the end doesn't it?

Reply:
Where are you thinking the semicolon goes, there is one after the do_query close bracket?  From previous examples of do_query, a semicolon is not needed before the end quotes.  Note, you should be able to see similar update query in function update_5_12_2004.

Reply:
Ok - I was under the impression that mysql required a semi-colon at the end of statements, but if you've tested it and it works then clearly it is fine.  I retract my question.